index.html

```diff
@@ -51,11 +51,11 @@ <h1 class="headline-1-h1">I’m Mariane
         </div>
         <div class="headline-4">
             <ul class="headline-4-ul">
-                <li><a href="#"><img alt="" src="icons/Vector1.svg"></a></li>
-                <li><a href="#"><img alt="" src="icons/Linkedin icon.svg"></a></li>
-                <li><a href="#"><img alt="" src="icons/Vector3.svg"></a></li>
-                <li><a href="#"><img alt="" src="icons/Vector2.svg"></a></li>
-                <li><a href="#"><img alt="" src="icons/Vector4.svg"></a></li>
+                <li><a href="#"><img alt="social media icon" src="icons/Vector1.svg"></a></li>
+                <li><a href="#"><img alt="social media icon" src="icons/Linkedin icon.svg"></a></li>
+                <li><a href="#"><img alt="social media icon" src="icons/Vector3.svg"></a></li>
+                <li><a href="#"><img alt="social media icon" src="icons/Vector2.svg"></a></li>
+                <li><a href="#"><img alt="social media icon" src="icons/Vector4.svg"></a></li>
```

Comment:
- [ ] Kindly make sure that the page has a minimum contrast ratio of at least 4.5:1. You can use [this aditus tool](https://www.aditus.io/button-contrast-checker/) to help to check the color contrast of the project.

![image](https://user-images.githubusercontent.com/62290545/221894072-0ebad2e3-6f86-4f79-adf2-86d11f599adf.png)
![image](https://user-images.githubusercontent.com/62290545/221894141-e1e92d37-d224-4acd-8d8f-2ce141c0097a.png)

Reply:
@ShadyShawkat 

> * [ ]  Kindly make sure that the page has a minimum contrast ratio of at least 4.5:1. You can use [this aditus tool](https://www.aditus.io/button-contrast-checker/) to help to check the color contrast of the project.
> 
> ![image](https://user-images.githubusercontent.com/62290545/221894072-0ebad2e3-6f86-4f79-adf2-86d11f599adf.png) ![image](https://user-images.githubusercontent.com/62290545/221894141-e1e92d37-d224-4acd-8d8f-2ce141c0097a.png)

@ShadyShawkat can you please explain how to fix this issue as we still don't understand the issue.

Reply:
@yin-ka Kindly read my suggestion for the answer.